setup.py

```diff
@@ -52,9 +26,17 @@ def readlines(filename):
         ],
     },
     setup_requires=["setuptools_scm"],
-    install_requires=install_deps,
+    install_requires=[
+        "dataclasses>=0.7; python_version < '3.7'",
+        "lxml>=4.0",
+        "skia-pathops>=0.4.1",
+    ],
     extras_require={
-        "dev": develop_deps,
+        "dev": [
+            "pytest",
+            "black==20.8b1",
+            "pytype==2020.11.23; python_version < '3.9'",
```

Comment:
I only kept black and pytype pinned to exact versions to avoid surprises with style suddenly formatting differently or typechecker being more or less picky all of a sudden. The dependabot should alert us of new releases for these.